src/block/block_ckpt.c

```diff
@@ -98,10 +98,7 @@ __wt_block_checkpoint_load(WT_SESSION_IMPL *session, WT_BLOCK *block, const uint
               block, &endp, ci->root_logid, ci->root_offset, ci->root_size, ci->root_checksum));
             *root_addr_sizep = WT_PTRDIFF(endp, root_addr);
 
-            if (block->log_structured) {
-                block->logid = ci->root_logid;
-                WT_ERR(__wt_block_newfile(session, block));
-            }
+            WT_ERR(__wt_block_tiered_load(session, block, ci));
```

Comment:
That's a very good point.  I'll add a TODO comment to tackle later, as I really want to get this merged before the changes become stale.

Reply:
Works for me.  Do you think we should have a ticket for it, too?

Reply:
I opened several tickets for TODOs WT-7470, WT-7471 (this one), WT-7472, WT-7473.  The last one is a "catch-all" for "TODO: tiered" comments that aren't otherwise tagged.

Reply:
Do we need all these TODOs regarding checkpoint since this will not be associated with checkpoint very long?